docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 36     | est_motor_amps                | get            | float     | A        | Estimated motor amps                                                                                                                 |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 37     | est_motor_torque              | get            | float     | Nm       | Estimated motor torque                                                                                                               |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 38     | motor_redline_start           | get, set, save | float     | rad/s    | Speed at which motor begins to derate                                                                                                |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 39     | motor_redline_end             | get, set, save | float     | rad/s    | Speed at which the motor is fully derated                                                                                            |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 40     | motor_l                       | get, set, save | float     | H        | Cross inductance                                                                                                                     |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 41     | derate                        | get            | int32     | PU fix16 | Amount of derating. No derate = 65536, full derate = 0                                                                               |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 44     | emf                           | get            | float     |          |                                                                                                                                      |      |
```

Comment:
Probably should add some notes on what all of these things actually do, otherwise its pretty inscrutable for customers. Me/Matt can help you with notes for them

Reply:
I agree.